src/components/views/settings/tabs/room/BridgeSettingsTab.js

```diff
@@ -0,0 +1,171 @@
+import MatrixClientPeg from "../../../../../MatrixClientPeg";
+import Pill from "../../../elements/Pill";
+import {makeUserPermalink} from "../../../../../utils/permalinks/Permalinks";
+import BaseAvatar from "../../../avatars/BaseAvatar";
+import { ContentRepo } from "matrix-js-sdk";
+
+const BRIDGE_EVENT_TYPES = [
+    "uk.half-shot.bridge",
+    // m.bridge
+];
+
+export default class BridgeSettingsTab extends React.Component {
+    static propTypes = {
+        roomId: PropTypes.string.isRequired,
+    };
+
+    constructor() {
+        super();
+
+        this.state = {
+        };
+    }
+
+    componentWillMount() {
+
+    }
+
+    _renderBridgeCard(event, room) {
+        const content = event.getContent();
+        if (!content || !content.channel || !content.protocol) {
+            return null;
+        }
+        const { channel, network } = content;
+        const protocolName = content.protocol.displayname || content.protocol.id;
+        const channelName = channel.displayname || channel.id;
+        const networkName = network ? network.displayname || network.id : protocolName;
+
+        let creator = null;
+        if (content.creator) {
+            const pill = <Pill
+                type={Pill.TYPE_USER_MENTION}
+                room={room}
+                url={makeUserPermalink(content.creator)}
+                shouldShowPillAvatar={true}
+            />;
+            creator = (<p>{
+                _t("This bridge was provisioned by %(pill)s", {
+                    pill,
+                })
+            } </p>);
+        }
+
+        const bot = (<p> {_t("This bridge is managed by the %(pill)s bot user.", {
+            pill: <Pill
+                type={Pill.TYPE_USER_MENTION}
+                room={room}
+                url={makeUserPermalink(event.getSender())}
+                shouldShowPillAvatar={true}
+                />,
+        })} </p>);
+        let channelLink = channelName;
+        if (channel.external_url) {
+            channelLink = <a target="_blank" href={channel.external_url}>{channelName}</a>;
+        }
+
+        let networkLink = networkName;
+        if (network && network.external_url) {
+            networkLink = <a target="_blank" href={network.external_url}>{networkName}</a>;
+        }
+
+        const chanAndNetworkInfo = (
+            (_t("Bridged into %(channelLink)s %(networkLink)s, on %(protocolName)s", {
+                channelLink,
+                networkLink,
+                protocolName,
+            }))
+        );
+
+        let networkIcon = null;
+        if (networkName && network.avatar) {
+            const avatarUrl = ContentRepo.getHttpUriForMxc(
+                MatrixClientPeg.get().getHomeserverUrl(),
+                network.avatar, 32, 32, "crop",
+            );
+            networkIcon = <BaseAvatar width={32} height={32} resizeMethod='crop'
+                            name={ networkName } idName={ networkName }
```

Comment:
It's super annoying imo, but we've decided that properties are to be indented 4 spaces, not aligned.